ppl/src/main/java/org/opensearch/sql/ppl/parser/AstBuilder.java

```diff
@@ -527,6 +528,34 @@ public UnresolvedPlan visitParseCommand(OpenSearchPPLParser.ParseCommandContext
     return new Parse(ParseMethod.REGEX, sourceField, pattern, ImmutableMap.of());
   }
 
+  @Override
+  public UnresolvedPlan visitSpathCommand(OpenSearchPPLParser.SpathCommandContext ctx) {
+    String inField = null;
+    String outField = null;
+    String path = null;
+
+    for (OpenSearchPPLParser.SpathParameterContext param : ctx.spathParameter()) {
```

Comment:
If an argument appears multiple times, we overwrite it with the new value.

This is consistent with how the other commands behave under the same scenario.

Reply:
Even input parameter can be multiple? I think it may be worth adding this comment in code.